### ISSUE TITLE
Crashes when trying to put down basket

### ISSUE THREAD

**Issue by sausagelinda** [copygirl/CarryCapacity#9]
using 5.2 but this also happened a few days ago using 5.1 

```
Running on 64 bit Windows with 16 GB RAM 
Version: v1.14.0 (Stable)12/12/2020 8:33:42 PM: Critical error occurred
System.MissingMethodException: Method not found: 'Void Vintagestory.API.Common.BlockEntity.FromTreeAtributes(Vintagestory.API.Datastructures.ITreeAttribute, Vintagestory.API.Common.IWorldAccessor)'.
   at CarryCapacity.CarriedBlock.RestoreBlockEntityData(IWorldAccessor world, BlockPos pos)
   at CarryCapacity.CarriedBlock.PlaceDown(IWorldAccessor world, BlockSelection selection, Entity entity)
   at CarryCapacity.Common.CarryHandler.OnGameTick(Single deltaTime)
   at Vintagestory.Common.EventManager.TriggerGameTick(Int64 ellapsedMilliseconds, IWorldAccessor world)
   at Vintagestory.Client.NoObf.ClientMain.MainRenderLoop(Single dt)
   at Vintagestory.Client.NoObf.ClientMain.MainGameLoop(Single deltaTime)
   at _6HHEy0R7TdcydZEq9IGp6JeHxdi._ogIdn3jhm7KUUuyvw6LDaa2X3PC(Single )
   at _gVBHOlByVD3vztgY4vPBxj2Bdae._ZKkvVIgccAdBc9j9BIZLqcgTrNL(Single )
   at _gVBHOlByVD3vztgY4vPBxj2Bdae._UQuOSpN5p2QIPbIExoPtD2ecXtb(Single )
   at Vintagestory.Client.NoObf.ClientPlatformWindows.window_RenderFrame(Object sender, FrameEventArgs e)
   at System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   at OpenTK.GameWindow.RaiseRenderFrame(Double elapsed, Double& timestamp)
   at OpenTK.GameWindow.DispatchRenderFrame()
   at OpenTK.GameWindow.Run(Double updates_per_second, Double frames_per_second)
   at _93RBbYKdbejiH9hVmbTS8c6kZxJ._hbMghNVJIKTGI5zBx3SuxRG1XDI(_Lsm9ms4vr0dfuNPxkLwK9w8a58N , String[] )
   at _eVFDVqwXngafMmcD9boVPO86iWx._hbMghNVJIKTGI5zBx3SuxRG1XDI(ThreadStart )
-------------------------------
```

**Comment by copygirl** [comment]
This is the error that was supposed to be fixed in v0.5.2, are you sure you correctly updated the mod? Does it show up with that version in the mod manager, and there are no duplicates?

**Comment by copygirl** [comment]
To anyone receiving this error, I hope you can help me out:

- Ensure you have version "v1.14.2 (Stable)" installed, you can see this in the bottom right of the main menu.
- Ensure you have the CarryCapacity mod installed, version 0.5.2, and only it. To make sure this is the case, open the mod manager ingame, click "Open Mods Folder", go one folder up and open `Cache`, then `unpack`. You should only see a folder called `CarryCapacity-1.14-0.5.2.zip_0404d3daf255`.
- Go to the game's install folder, and send me the "VintagestoryAPI.dll" - I would like to compare it to mine, see if there's any difference between them.
